src/Microsoft.PowerShell.ConsoleHost/host/msh/ProgressPane.cs

```diff
@@ -183,9 +237,21 @@ class ProgressPane
                 return;
             }
 
-            // NTRAID#Windows OS Bugs-1061752-2004/12/15-sburns should read a skin setting here...
+            BufferCell[,] newRegion;
+            if (ProgressNode.IsMinimalProgressRenderingEnabled())
+            {
+                // Legacy progress rendering relies on a BufferCell which defines a character, foreground color, and background color
+                // per cell.  This model doesn't work with ANSI escape sequences.  However, there is existing logic on rendering that
+                // relies on the existence of the BufferCell to know if something has been rendered previously.  Here we are creating
+                // an empty BufferCell, but using the second dimension to capture the number of rows so that we can clear that many
+                // elsewhere in Hide().
+                newRegion = new BufferCell[0, _content.Length];
```

Comment:
If it is not used can it be a cached and reused object?

Reply:
The object will change if there are child or sibling progress objects.